app/styles/identity.css

```diff
@@ -154,3 +156,299 @@
     width: 100%;
   }
 }
+
+/* New CSS */
+
+.identity-page {
+  height: 91.5vh;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  /* justify-content: center; */
+  align-items: center;
+}
+
+.identity-hero {
+  position: absolute;
+  left: 0;
+  bottom: 0;
+  z-index: 1;
+}
+
+.identity-heading {
+  font-family: 'Hanuman', serif;
+  color: #000;
```

Comment:
Use css variable  here

Reply:
I don't think there is a need to put the black color in the variable.